app/Models/Campaign.php

```diff
@@ -370,6 +370,14 @@ public function defaultToConnection(): bool
         return (bool) Arr::get($this->ui_settings, 'connections', false);
     }
 
+    /**
+     * @return mixed
+     */
+    public function defaultToConnectionMode(): int
```

Comment:
I feel like the fallback from Arr::get should be an int rather than a book then cast to an int, makes the code easier to read :)